.github/workflows/ci.yml

```diff
@@ -0,0 +1,36 @@
+name: Codecov
+on:
+  push:
+    branches: [ git_workflow ]
```

Comment:
This will be updated right?

Reply:
we have to trigger this on publish so i will change it to 

```
on: 
    release:
       types: [published]
```